src/bp/common/validation.ts

```diff
@@ -22,6 +23,7 @@ export const BotEditSchema = Joi.object().keys({
     .min(3)
     .max(50)
     .required(),
+  category: Joi.string(),
```

Comment:
Category should be optional

Reply:
it is by default when not marked as required if i recall